tests/Microsoft.Bot.Connector.Tests/Authentication/UserTokenClientImplTests.cs

```diff
@@ -0,0 +1,309 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Bot.Connector.Authentication;
+using Microsoft.Bot.Schema;
+using Microsoft.Rest;
+using Xunit;
+
+namespace Microsoft.Bot.Connector.Tests.Authentication
+{
+    public class UserTokenClientImplTests
+    {
+        private const string AppId = "test-app-id";
+        private const string ToChannelFromBotOAuthScope = "oauth-scope";
```

Comment:
This constant is not being used, it can be removed.

Reply:
removed, ty